docs/xss.md

```diff
@@ -24,14 +24,13 @@
 
 # -Xiss / -Xss / -Xssi
 
-Determines the size and behavior of the stack size for Java&trade; threads. 
-
-To increase the maximum number of threads your system can support, reduce the maximum native stack size.
+Determines the size and behavior of the stack size for Java&trade; threads.
 
 If you exceed the maximum Java thread stack size, a `java/lang/OutOfMemoryError` message is reported.
 
 You can use the `-verbose:sizes` option to find out the values that the VM is currently using.
 
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** These options relate to the Java stack only. There is a separate, native stack for operating system threads, for which you use a different option (see the link in the [See also](#see-also) section).
```

Comment:
Same comment about 2 stacks here

Reply:
A lot of the doc currently talks about java threads and native threads as separate entities. For example, in https://www.eclipse.org/openj9/docs/dump_javadump/#threads we say "Java threads, native threads, and stack traces" and even "A Java thread runs on a native thread." (We used to say "Each native thread is paired with the corresponding Java thread, if one exists.", but not anymore it seems!) Not sure if these are just different ways of looking at the same thing.

Reply:
Made the suggested update, thanks